### DISCUSSION TITLE
Pass --no-verify flag to git commit when moving patches into new commits

### DISCUSSION THREAD

**Discussion by juru1234** [jesseduffield/lazygit#2995] [Q&A]
Hi, is there a way to use `git commit --no-verify` when moving a patch into a new commit and lazygit calls "git commit" indirectly?

**Comment by stefanhaller** [comment]
No. I'm unsure what UX we could imagine for making that possible. A config to enable `--no-verify` always in this situation? I'm unsure how useful that would be.

But it seems that we don't respect the `skipHookPrefix` config when moving a patch into a new commit; neither for the commit that we are moving from, nor for the new commit. The former also applies to move patch into index. This just feels like a bug. Would that help you already?

**Reply by juru1234** [reply]
I don't think that it is necessary to change the UX for that. It would just be nice if the `skipHookPrefix` could be respected in that case :)

**Reply by juru1234** [reply]
Should I open an issue for that?

**Reply by stefanhaller** [reply]
Yes please!

**Reply by golergka** [reply]
Personally, I dislike staging and prefer to do temporary "WIP" commits instead. However, when using a repo with pre-commit hooks, this workflow makes me switch to command line all the time.

Would you mind if I tried to implement it? I want to experiment with various UX solutions. (I'm asking just in case you don't want this feature in Lazgit at all, then I'd save myself time doing work which will never be used).

**Reply by stefanhaller** [reply]
@golergka I'm not quite sure what exactly you are talking about when you say "implement it" and "this feature". Which feature exactly? In this issue we are talking about the bug that the skipHookPrefix config is not always respected, e.g. when moving a patch into a new commit. Is that what you are talking about? There is already a PR open for that: #3532. (@mark2185, any chance to make some progress with that at some point? 😉)

**Reply by golergka** [reply]
> I'm not quite sure what exactly you are talking about when you say "implement it" and "this feature". Which feature exactly?

Sorry, I must have gotten confused here. I searching for an existing issue for an improvement I want to implement, and somehow missed that this one talked about moving patches, not making commits in general. I'll create a separate discussion
